packages/map/src/CdcMap.tsx

```diff
@@ -831,6 +831,20 @@ const CdcMap = ({
 
     legendMemo.current = newLegendMemo
 
+    if (state.general.geoType === 'world') {
+      const runtimeDataKeys = Object.keys(runtimeData)
+      const isCountriesWithNoDataState =
+        obj.data === undefined ? false : !countryKeys.every(countryKey => runtimeDataKeys.includes(countryKey))
+
+      if (result.length > 0 && isCountriesWithNoDataState) {
+        result.push({
+          min: null,
+          max: null,
+          color: '#E6E6E6'
```

Comment:
The default background color has changed to --coolGray10. You should be able to get that using the helper function `getGeoFillColor` found in `/map/helpers/colors.ts`

Reply:
@adamdoe  I am trying to get this to work. I am not understanding how getGeoFillColor should work. I keep getting a black background.

Reply:
Hi @Matt-DiPaolo - Josh made changes earlier today. You can check his commit for the updates if you want to review them.